site/src/comparison.rs

```diff
@@ -818,14 +843,22 @@ impl TestResultComparison {
         } else {
             self.variance
                 .as_ref()
-                .map(|s| s.upper_fence())
+                .map(|v| v.significance_threshold())
                 .unwrap_or(Self::SIGNIFICANT_RELATIVE_CHANGE_THRESHOLD)
         }
     }
 
+    /// This is a numeric magnitude of a particular change.
+    fn significance_factor(&self) -> Option<f64> {
```

Comment:
What's the reason for this being an optional?

Reply:
Hm, I think this was leftover from an earlier version I had locally. The reasoning was that we may not always be able to compute a change's magnitude in a reasonable way -- we approximate significance with the 0.002 threshold constants, but there's no such thing really for "how significant" if there's no prior data.

I actually think this may be a little buggy in the sense that I think we should show something like "-" in the UI if we don't have enough historical data to judge the factor, not just scale based on the 0.002 threshold, which would be misleading.